doc_src/cmds/fish_status_to_signal.rst

```diff
@@ -0,0 +1,29 @@
+.. _cmd-fish_status_to_signal:
+
+fish_status_to_signal - Convert exit codes to human-friendly signals
+====================================================================
+
+Synopsis
+--------
+
+::
+
+    function fish_prompt
+        echo -n (fish_status_to_signal $pipestatus) (prompt_pwd) '$ '
+    end
+
+Description
+-----------
+
+``fish_status_to_signal`` converts exit codes to their corresponding human-friendly signals if one exists.
+This is likely to be useful for prompts in conjunction with the ``$status`` and ``$pipestatus`` variables.
```

Comment:
It's probably useful to note that this is a *guess* - there's nothing that enforces that e.g. status 129 has to be caused by a SIGHUP, a program could just decide to return it itself.

(AFAIK the mapping of signal to status is very common - it might even be guaranteed by the standard)